src/Backend/API.ts

```diff
@@ -44,15 +44,11 @@ function backendRegistrationApi() {
       if (compiler) {
         gToolchainEnvMap[backend.name()] = new ToolchainEnv(compiler);
       }
-      const executor = backend.executor();
-      if (executor) {
-        globalExecutorArray.push(executor);
+      const executors = backend.executors();
+      if (executors) {
+        globalExecutorArray.push(...executors);
```

Comment:
This array could be removed. but current PR not removed. this will remove next PR about `DeviceView`.

Reply:
Q) Is it ready to modify our backend extension repo? 
IMHO Revisiting inference and execute quickinputs and runner doesn't matter for the other developers, but failing to activate extensions could be critical. 
How about remaining this part as is, and just adding `executors` to `Backend.ts`? Since renaming to `executors` causes error while activating the extensions. It might be short time, but someone who rebased main branch will be confused from the error.

Reply:
Hmm.. I thought these lines could be removed, but you commented `this will remove next PR about DeviceView`. I think I couldn't clear understand what that means. Can you give me a little more details?

Reply:
rr> `How about remaining this part as is, and just adding executors to Backend.ts?` 

I agree here.

More specifically,
You can (1) first add 'executors' and, (2) replace the internal logic of `executor` with that of `executors`, add 'TODO Remove executor' comment, (3) then finally remove executor.

Reply:
> Hmm.. I thought these lines could be removed, but you commented `this will remove next PR about DeviceView`. I think I couldn't clear understand what that means. Can you give me a little more details?

Oh, I write this to remind me to check this. but I think this make you more confused. I will edit this comment as

```
This array could be removed as we can access all `Executor`s using `globalBackendMap`
```

Sorry for make you confusing!

Reply:
> More specifically,
> You can (1) first add 'executors' and, (2) replace the internal logic of `executor` with that of `executors`, add 'TODO Remove executor' comment, (3) then finally remove executor.

If so, I think I will make a new PR and remain this as DRAFT.